integration-tests/src/test/resources/domain-home-on-pv/jrf/create-domain-script.sh

```diff
@@ -0,0 +1,200 @@
+#!/bin/bash
```

Comment:
This WLST is based on the old OracleFMWInfrastructure image.

Reply:
Specifically, you need to please catch up with changes made here: https://github.com/oracle/docker-images/blob/master/OracleFMWInfrastructure/dockerfiles/12.2.1.3/container-scripts/createInfraDomain.py

Reply:
pushed another commit to use the new fmw infra image. Please review. Thanks

Reply:
Mark is trying to push a JRF sample ( like our current WLS sample)  into develop branch from jrf-support branch which will provide an e-2-e sample to bring JRF domain in Operator Environment.  After the availability of the jrf sample, we need to modify the integration-test infra to use the sample scripts to bring up the JRF domain

Reply:
@anpanigr yes, we need to update the test once Mark's PR is merged to develop branch.

Reply:
@mriccell  @rjeberhard @anpanigr The test is updated to use Mark's new script to create the fmw infra domain and use the new OracleFMWInfrastructure image. Please review it again. Thanks!